code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -75,11 +76,10 @@ GLOBAL_LIST_EMPTY(parasites) //all currently existing/living guardians
 	GLOB.parasites -= src
 	return ..()
 
-/mob/living/simple_animal/hostile/guardian/proc/setthemename(pickedtheme) //set the guardian's theme to something cool!
-	if(!pickedtheme)
-		pickedtheme = pick("magic", "tech", "carp")
+/mob/living/simple_animal/hostile/guardian/proc/setthemename() //set the guardian's theme to something cool!
```

Comment:
yeah why did you change this at all? that wasn't the instruction at all.

set the current_theme in the update_theme proc, from the passed in value